client.go

```diff
@@ -55,8 +92,6 @@ func (c *Client) sendRequest(req *http.Request, v any) error {
 		req.Header.Set("Content-Type", "application/json; charset=utf-8")
 	}
 
-	c.setCommonHeaders(req)
-
 	res, err := c.config.HTTPClient.Do(req)
 	if err != nil {
 		return err
```

Comment:
I wanted to cover the above line.
https://github.com/sashabaranov/go-openai/pull/395#discussion_r1237604960